source/reference/files/submit-yml/script-options.rst

```diff
@@ -0,0 +1,460 @@
+.. _script-options:
+
+Script Options
+==============
```

Comment:
Instead of documenting this here, add any appropriate example documentation inline in the code for ood_core and link to generated yardoc for ood_core from here.

Currently this is available from https://osc.github.io/ood_core/ but we are not auto-generating that yet for each - a travis ci or github actions needs added to do that...

Reply:
https://osc.github.io/ood_core/0.11.4/OodCore/Job/Script.html

The point is the overview is just one string of text:

>An object that describes a batch job before it is submitted. This includes the resources this batch job will require of the resource manager.

But it could be detailed description of the use of the batch connect yaml how it corresponds to the defined methods. Or enough information so that each method documentation provides its own example of using via the batch connect yaml.

Reply:
Won't that be confusing to users to have to interpret ruby docs?  I just mean they'll be unfamiliar with it and it may not be easy to read for them.

Reply:
I guess it depends on how the Ruby docs are formatted. If there is enough explanation above it probably isn't that confusing. FWIW we could also use the yard tool to generate a restructured version of this documentation and then use that in here. Adding directions to help address any confusion of "intepretting ruby docs" is easiest though.

Reply:
Maybe we can just get rid of this page so the PR can move forward? I can then create follow up tickets to (a) create a tasks to auto publish the yarn docs and (b) add a link to that here.

Reply:
Sounds good